ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml

```diff
@@ -1,8 +1,8 @@
 base_images:
   os:
```

Comment:
think this should be krkn to be consistent with what the base image is

Reply:
Done!